src/pages/home/report/ReportActionItemCreated.tsx

```diff
@@ -0,0 +1,129 @@
+import React, {memo} from 'react';
+import {View} from 'react-native';
+import type {OnyxEntry} from 'react-native-onyx';
+import {withOnyx} from 'react-native-onyx';
+import MultipleAvatars from '@components/MultipleAvatars';
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import PressableWithoutFeedback from '@components/Pressable/PressableWithoutFeedback';
+import ReportWelcomeText from '@components/ReportWelcomeText';
+import useLocalize from '@hooks/useLocalize';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import reportWithoutHasDraftSelector from '@libs/OnyxSelectors/reportWithoutHasDraftSelector';
+import * as ReportUtils from '@libs/ReportUtils';
+import {navigateToConciergeChatAndDeleteReport} from '@userActions/Report';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {PersonalDetailsList, Policy, Report} from '@src/types/onyx';
+import AnimatedEmptyStateBackground from './AnimatedEmptyStateBackground';
+
+type OnyxProps = {
+    /** The report currently being looked at */
+    report: OnyxEntry<Report>;
+
+    /** The policy being used */
+    policy: OnyxEntry<Policy>;
+
+    /** Personal details of all the users */
+    personalDetails: OnyxEntry<PersonalDetailsList>;
+};
+
+type ReportActionItemCreatedProps = OnyxProps & {
+    /** The id of the report */
+    reportID: string;
+
+    /** The id of the policy */
+    // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
Any idea how we can remove the `policyID`?

It is used anywhere in the component.

Reply:
Unfortunately no, `policyID` is used in the HOC, that's why we need it here.

Reply:
Thanks